webrtc/RTCConfiguration-rtcpMuxPolicy.html

```diff
@@ -140,4 +140,56 @@
       Tested    2
       Total     2
    */
+  const FINGERPRINT_SHA256 = '00:00:00:00:00:00:00:00:00:00:00:00:00' +
+      ':00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00';
+  const ICEUFRAG = 'someufrag';
+  const ICEPWD = 'somelongpwdwithenoughrandomness';
+
+  promise_test(async t => {
+    const sdp = 'v=0\r\n' +
```

Comment:
main reason for static SDP is that bundle implies rtcp-mux so we need to do unbundled